brianlong/www.validators.app
# 180879183 stake pool metrics explained

#### What's this PR do?
- Add explanation of stake pool metrics below the table

#### How should this be manually tested?
- go to /stake-pools and scroll to bottom of the page. Check for grammar and meritoric mistakes

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/180879183)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
